app/models/beer.rb

```diff
@@ -24,4 +24,12 @@ def self.search_message
       "No Matching Beer"
     end
   end
+
+  def average_rating
+    if reviews.average(:rating) == nil
+      reviews.average(:rating)
+    else
+      reviews.average(:rating).round(1)
```

Comment:
Do the rounding in the view. What if your client would want 2 decimals later or 0? It makes it easier to pinpoint and change.